app/models/partner.rb

```diff
@@ -32,4 +32,12 @@ def confirmed?
   def full_name
     name
   end
+
+  def to_csv
+    columns = %w[created_at updated_at email name phone_number]
+    CSV.generate(headers: true) do |csv|
+      csv << columns
+      csv << columns.map { |column| send(column) }
```

Comment:
Il existe la methode `public_send` pour eval dynamiquement une méthode et retourner une erreur si celle-ci est privée.
À moins que tu cherches explicitement à appeler une méthode privée via `send`, je t'invite à utiliser `public_send` par défaut 🙂 

```suggestion
      csv << columns.map { |column| public_send(column) }
```